doc/BUILD.rst

```diff
@@ -369,7 +364,7 @@ Deprecation Warnings (``-Ddeprecation_warnings``)
 * Default: ``default``
 * Options: ``error``, ``no-error``, ``ignore``, ``default``
 
-Sets deprecation warnings to either hard-fail (``error``), print as warnings but
+Sets deprecation warnings to either hard-fail (``default``/```error``), print as warnings but
```

Comment:
`default` and `error` aren't equivalent: `error` makes them errors regardless of whether `werror` is enabled or not, while `default` respects the `werror` setting.